### PR TITLE
openems: unstable-2020-02-15 -> 0.0.36 

### LINKED ISSUE #266953: Update request: openems 2020-02-15 → 0.0.36
- Package name: OpenEMS
- Latest released version: 0.0.36

- Current version on the unstable channel: 2020-02-15
- Current version on the stable/release channel: 2020-02-15

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@matthuszagh 
-----

Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by mayl**
## Description of changes

[release announcement](https://github.com/thliebig/openEMS-Project/discussions/132)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Resolves #266953

### CODE REVIEW

**Comment by mayl** [comment]
Tested this with the patch antenna example from upstream successfully.

I updated `openems` `python-openems` `csxcad` and `python-csxcad` all at once since they seem like they are pretty coupled to one another to work correctly...

File: pkgs/applications/science/electronics/csxcad/default.nix [hunk 1]
```diff
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "csxcad";
-  version = "unstable-2022-05-18";
+  version = "v0.6.3";
```
Comment (pbsds) [inline pkgs/applications/science/electronics/csxcad/default.nix:16]:
```suggestion
  version = "0.6.3";
```

File: pkgs/applications/science/electronics/openems/default.nix [hunk 2]
```diff
@@ -20,13 +20,13 @@
 
 stdenv.mkDerivation {
   pname = "openems";
-  version = "unstable-2020-02-15";
+  version = "v0.0.36";
```
Comment (pbsds) [inline pkgs/applications/science/electronics/openems/default.nix:23]:
```suggestion
  version = "0.0.36";
```

File: pkgs/applications/science/electronics/openems/default.nix [hunk 3]
```diff
@@ -20,13 +20,13 @@
...
 
   src = fetchFromGitHub {
     owner = "thliebig";
     repo = "openEMS";
-    rev = "ba793ac84e2f78f254d6d690bb5a4c626326bbfd";
-    sha256 = "1dca6b6ccy771irxzsj075zvpa3dlzv4mjb8xyg9d889dqlgyl45";
+    rev = "v0.0.36";
```
Comment (pbsds) [inline pkgs/applications/science/electronics/openems/default.nix:28]:
```suggestion
    rev = "v${version}";
```

**Review by pbsds — COMMENTED** [review]
sorry, should've caught this the first round

**Comment by mayl** [comment]
np, pushed new changes

**Comment by mayl** [comment]
Whoops, needed to make openems attrs recursive to expression the version the way you suggested.  Should work now...

**Comment by mayl** [comment]
Ugh.  Sorry, I misstyped the interpolation syntax.  I'm doing this in between other things and obviously need to slow down...

I manually built these on my machine, so this should be working again now...  :crossed_fingers:

**Pull request merged**